core/config/mainnet.json

```diff
@@ -9,7 +9,7 @@
             {
                 "name": "Homestead",
                 "block": 1150000,
-                "requiredHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+                "requiredHash": "0x584bdb5d4e74fe97f5a5222b533fe1322fd0b6ad3eb03f02c3221984e2c0b430",
```

Comment:
This PR includes the changes from https://github.com/ethereumproject/go-ethereum/pull/612. We can extract these changes if the decision is negative on the concept. The changes to `core/config/assets.go` are also from this.